### PR TITLE
docs: edit readme

### PR DESCRIPTION
**Pull request by matgorzynski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+  - [Contributing](#contributing)
+  - [License](#license)
+  - [Contact](#contact)
+
+## Introduction
+
+Cuisine Trove is a partially open source online cookbook project. Our project is set up so that you can run it locally on your own machine. The service is also available at: somewebsite.com
+
+## Features
+- Search recipe: by name or ingredients
```
Comment (adr7dev) [inline README.md:18]: "Search for a recipe"

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+- Management of recipes
```
Comment (adr7dev) [inline README.md:19]: correct but "Recipes management" sounds more compact

File: README.md (hunk 1 above)
Comment (matgorzynski) [reply README.md]: comment suggestion applied

File: README.md (hunk 2 above)
Comment (matgorzynski) [reply README.md]: comment suggestion applied

**Review by adr7dev — APPROVED** [review]

**Pull request merged**